deployment_scripts/deploy_linux.sh

```diff
@@ -101,25 +101,25 @@ log_message "Creating island dirs under $ISLAND_PATH"
 mkdir -p "${MONGO_PATH}" || handle_error
 mkdir -p "${ISLAND_BINARIES_PATH}" || handle_error
 
-# Detecting command that calls python 3.7
+# Detecting command that calls python 3.11.2
```

Comment:
Do we absolutely need 3.11.2? Won't 3.11.x work?

Reply:
I am not sure. I think 3.11.2 is the latest one so we chose that.

Reply:
I think we need 3.11.2

Reply:
To be more specific we need 3.11.^2. But asserting this is harder, so I think testing 3.11.* is good enough